src/metadatas.rs

```diff
@@ -163,3 +180,19 @@ fn test_stop_need_phone_driver() {
     assert!(!metadatas.some_stops_need_phone_agency);
     assert!(metadatas.some_stops_need_phone_driver);
 }
+
+#[test]
+fn test_route_have_colors() {
+    let raw_gtfs =
+        gtfs_structures::RawGtfs::new("test_data/no_fares_no_shapes").expect("Failed to load data");
+    let metadatas = extract_metadata(&raw_gtfs);
+    assert_eq!(2, metadatas.lines_with_custom_color_count);
+}
+
+#[test]
+fn test_route_no_color() {
```

Comment:
The test name is weird because you have 3 custom colors.

Reply:
Thanks, I changed. I didn't update the test name after @Tristramg feedback.